packages/gestalt/src/TapArea.js

```diff
@@ -218,6 +227,7 @@ const TapAreaWithForwardRef: AbstractComponent<Props, HTMLDivElement> = forwardR
       [styles.fullHeight]: fullHeight,
       [styles.fullWidth]: fullWidth,
       [styles.copy]: mouseCursor === 'copy' && !disabled,
+      [styles.default]: mouseCursor === 'default' && !disabled,
```

Comment:
&& !disabled, is kind of redundant no? 

`      [styles.default]: mouseCursor === 'default' || disabled,
`
no?